src/model/SeqCollection.js

```diff
@@ -43,7 +43,10 @@ const SeqCollection = Collection.extend({
   // true: returns the last element, false: returns undefined
   prev: function(model, endless) {
     let index = this.indexOf(model) - 1;
-    if (index < 0 && endless) { index = this.length - 1; }
+    if (index < 0) { 
+      if (endless) { index = this.length - 1; }
+      else { return null; }
+    }
     return this.at(index);
```

Comment:
This was responsible for the bug where the borders disappeared for the residues in the first row (if the same-indexed residues were selected in the last row). It was because the `prev` function was returning the last row as the prev row for the first (because `this.at(-1)` is a defined operation where the last element of the array is returned).